components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -4886,6 +4886,130 @@ document.observe("xwiki:dom:loaded", function() {
   </object>
   <object>
     <name>PhenoTips.Widgets</name>
+    <number>14</number>
```

Comment:
Since JS extensions are executed in the order they are imported, we should not have this as a #14 widget since this is the first thing a user sees (sections collapsing/expanding), and which shoul dbe done ASAP to improve user experience

Instead we should place it into a separate document, and import first before all widgets and before PatientSheetCode (or FamilySheetCode)

I wanted to do this for a long time just had other things to do - since you started, maybe you can do that please?